core/server/errors.js

```diff
@@ -29,6 +29,7 @@ function GhostError(options) {
     this.help = options.help || this.help;
     this.errorType = this.name = options.errorType || this.errorType;
     this.errorDetails = options.errorDetails;
+    this.code = options.code || null;
```

Comment:
:+1: Yes! See #6526 - I think this is great. 
Error Code: either the Node code, or an identifier which differentiates instances of an errorType (exactly how you used this)
Error UID: an ID for each individual error that is thrown